File: Intro-to-Programming-Language/Module-3/even_odd_sum.py
n = int(input("Enter number :"))

sum = 0
even_number_sum =0
odd_number_sum =0
for i in range(1,n+1):
    sum +=i
    if i%2==0:
        even_number_sum +=i
    else:
        odd_number_sum +=i
print("Sum : ",sum)
print("Even number of sum :", even_number_sum)
print("Odd number of sum : ",odd_number_sum)

File: Intro-to-Programming-Language/Module-3/even_odd_sum.c
#include<stdio.h>
int main(){
    int n;
    scanf("%d",&n);
    int even_sum =0, odd_sum=0;
    for(int i=1;i<=n;i++){
        if(i%2==0){
            even_sum +=i;
        }else{
            odd_sum +=i;
        }
    }
    printf("Sum of Even Number :%d\n",even_sum);
    printf("Sum of odd Number : %d\n",odd_sum);
    return 0;
}
// output
// n=5
// even sum = 0+2+4 =6
// odd sum = 1+3+5 = 9